src/Debug.js

```diff
@@ -44,7 +44,7 @@ function propsString(node) {
 
 export function debugNode(node, indentLength = 2) {
   if (!node) return '';
```

Comment:
While i'm looking at this, it seems like this should be
```
if (!node && node !== 0) return '';
```

Since react will stringify `0` to `'0'`